manifests/init.pp

```diff
@@ -10,11 +13,64 @@
 #   if false packages will be installed with "present"
 # @param  $version
 #   The version of snmp protocol to use.
-#   At this time the simp_profile only manages v3, to configure
+#   At this time the simp_snmpd profile only manages v3, to configure
 #   older versions use the snmp module directly.
+# @param  $snmp_conf_file,
+#   A file of snmp.conf directives that is included for configuration directives.
+#   this file is managed by puppet.
+# @param  $simp_snmpd_dir,
+#   a directory of *.conf files which include snmpd directives.  Files in this
+#   directory are managed by puppet.
```

Comment:
Is the directory managed?  In other words, will files not under puppet control be removed from this directory?